src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketTagClientImpl.java

```diff
@@ -101,10 +79,6 @@ static class NextPageFetcherImpl implements NextPageFetcher<BitbucketTag> {
 
         @Override
         public BitbucketPage<BitbucketTag> next(BitbucketPage<BitbucketTag> previous) {
-            if (previous.isLastPage()) {
```

Comment:
This gets picked up from the underlying implementation, no need to have it here. Just something i found while testing.